local-cli/generator-android/templates/src/app/react.gradle

```diff
@@ -32,19 +32,21 @@ gradle.projectsEvaluated {
         buildTypes.each { buildTypeName ->
             // Create variant and source names
             def sourceName = "${buildTypeName}"
-            def targetName = "${sourceName.capitalize()}"
+            def targetPath = "${buildTypeName}"
             if (productFlavorName) {
-                sourceName = "${productFlavorName}${targetName}"
+                sourceName = "${productFlavorName}${sourceName.capitalize()}"
```

Comment:
Won't this become the gradle task `assembleflavorDebug` when we want `assembleFlavorDebug`?